Doc: How we schedule meetings (#10)
hyphacoop/handbook: guides.md

# Guides

Guides explain in practical terms how we do stuff. Any Hypha member can
change these.<sup>[1](#references)</sup>

We are inspired by the idea of "protocols" from the [Lab
Book][clear-lab-book] of the [Civic Laboratory for Environmental Action
Research (CLEAR)][clear]

> Protocols are the "attitudes" or "the manner in which one approaches each and
every element in our space." They are different than pure rules or
instructions; they are ways that we establish order and maintain practices
across our group [...] -- They define the way we ought to proceed or behave in
different situations. As such, they are normative, or premised on values,
morals, and an idea of how things ought to be done. They are a manifestation of
our values.

## Scheduling a meeting

Note: Consider whether this meeting might be a global interest to other

> members. Skip below steps at your discretion for low-stakes topics.

Reviewer: ```suggestion
members. Skip the steps below at your discretion for low-stakes topics.
```

> - Try to announce your intention to schedule a meeting on a topic, and ask

Reviewer: ```suggestion
- Try to announce your intention to schedule a meeting on a topic and ask
```

>   who's interested to participate. Allow a day for people to respond.

Reviewer: ```suggestion
  who's interested in participating. Allow a day for people to respond.
```

>   who's interested in participating. Allow a day for people to respond.

Author: ```suggestion
  who's interested in participating. Allow _at least_ 48 hours for people to respond.
```

> - Consider who might be most interested, and what timing might work for

Reviewer: ```suggestion
- Consider who might be most interested and what timing might work for
```

  them.

>   - Check the [monthly `availability` calendar for travel/vacation plans][availability].

Author: ```suggestion
  - Check the [`availability` calendar for travel/vacation plans][availability].
```

>   - Check everyone's [weekly availability `schedules`][schedules].

Author: ```suggestion
  - Check everyone's [availability `schedules`][schedules].
```

> - If 2-3 candidate timeslots feel obvious from looking at internal

Author: ```suggestion
- If 2-3 candidate timeslots feel obvious from looking at people's
```

>   availabilities:

Reviewer: clarify what "internal availabilities" means (I'm really struggling), could we say "people's availability"?

Author: "internal availabilities" was just reference to one standing internal when2meet for members. i'm just trying to find language to avoid saying "look at when2meet"... which itself can be a "fresh" one, or using that standing internal one. As in, when scheduling with Jutta, I didn't blow open the scheduling to include everyone, but I checked our "internal availability" on that standing when2meet.

Is this worth capturing here? Happy to use whatever lingo, or even to just drop it as TMI :)

Author: Maybe these could use clearer lingo...

- "availability calendar" (gcal) vs "availability schedule" (when2meet)?

Author: dc and i spoke and sorted 🌮 #goodenoughtada

  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
    Infrastructure Meeting (Option 1)`
  - Ask in chat whether anyone who wants to attend has any blocks on any
    of the options.
- If short-notice or speed of booking is important:
  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
    - If time permits or topic isn't an all-hands matter, consider
      leaving the hold available for someone else.

>     - If claiming a hold, edit the calendar immediately, and announce

Reviewer: ```suggestion
    - If claiming a hold, edit the calendar immediately and announce
```

      meeting in chat.
- If your must find a new timeslot:

>   - Create a new poll. (We currently use [When2Meet](https://www.when2meet.com/).)

Reviewer: ```suggestion
  - Create a new poll (We currently use [When2Meet](https://www.when2meet.com/)).
```

Author: oh jeez, my intuitions on this stuff are different. I believe punctuation is fine to go inside when the whole sentence is parenthetical, right?
https://english.stackexchange.com/a/11129/55389

I'm less inclined to do it your suggested way, unless it's of the structure "(and ...)" or "(when ... )". But as it stands, it's a full sentence in the parens :)

Thoughts?

Reviewer: If it is a full sentence than it shouldn't be in a parenthetical, as I'm used to them explaining/qualifying and doing so while signalling it is set apart from of the flow of writing. This definition roughly corresponds to my understanding: https://www.vocabulary.com/dictionary/parenthetical

  - Leave ample time for completion (ideally 1 week).

>   - Check on non-responders in following days, and send (gentle)

Reviewer: ```suggestion
  - Check on non-responders in following days and send (gentle)
```

    reminders as needed.

> ## Hosting a meeting

Reviewer: See my point below about moving this -- would not be my preference

- **Before** the meeting...
  - create a stub agenda as soon as possible (can be done before scheduling)
    - Hint: use [an agenda `template`][template]
  - add the agenda to the [`meetings` index][meetings]

>   - update the [organizational calendar][calendar] as event scheduling

Author: ```suggestion
  - update the [organizational calendar][calendar] as early as possible with candidate and finalized time slots
```

    progresses
  - mention the meeting time in chat, and link the agenda.
  - encourage migration of related actions, decisions and discussions
    into agenda from chat
- **During** the meeting...
  - ask for agenda items (start of meeting)
  - facilitate discussion
  - keep time
  - call attention to action items, upcoming decisions and discussions
- **After** the meeting...
  - add important dates/reminders to calendar
  - review and merge notes
  - update [meeting index][meetings] as best location changes
    (hackmd => review (PR) => file)
  - as needed, set up loomio threads and decisions
  - as needed, send reminders of action items

>   - future: migrate action items into task tracker

Reviewer: Unclear how "future" is being used here, can we just get rid of "future" given task tracking immanent arrival?

Author: Good point :)

Author: ```suggestion
  - migrate action items into task tracker
```
